src/app/App.jsx

```diff
@@ -46,74 +48,65 @@ function App() {
   }
 
   return (
-    <div className="flex flex-col h-screen">
-      {user && (
-        <Header
-          user={user}
-          clickHandleLogout={setUser}
-          currentDBId={currentDBId}
-          isEditMode={isEditMode}
-          onClickSave={setIsEditMode}
-          currentDocIndex={currentDocIndex}
-          clickHandleNavigator={setCurrentDocIndex}
-        />
-      )}
-      <div className="flex flex-1">
-        {user && (
-          <Sidebar
-            user={user}
-            currentDBId={currentDBId}
-            setCurrentDBId={setCurrentDBId}
-            setDocumentsIds={setDocumentsIds}
-          />
-        )}
-        <div className="flex grow justify-center">
-          <Routes>
-            <Route path="/login" element={<Login setUser={setUser} />} />
-            <Route
-              path="/dashboard"
-              element={<ContentsContainer user={user} />}
-            >
-              <Route
-                path="listview"
-                element={
-                  <ListView
-                    user={user}
-                    currentDBId={currentDBId}
-                    isEditMode={isEditMode}
-                    setIsSaveMode={setIsEditMode}
-                    currentDocIndex={currentDocIndex}
-                    setCurrentDocIndex={setCurrentDocIndex}
-                    setDocumentsIds={setDocumentsIds}
-                  />
-                }
+    <UserContext.Provider value={user}>
+      <CurrentDBIdContext.Provider value={currentDBId}>
+        <div className="flex flex-col h-screen">
+          {user && (
+            <Header
+              clickHandleLogout={setUser}
+              isEditMode={isEditMode}
+              onClickSave={setIsEditMode}
+              currentDocIndex={currentDocIndex}
+              clickHandleNavigator={setCurrentDocIndex}
+            />
+          )}
+          <div className="flex flex-1">
+            {user && (
+              <Sidebar
+                setCurrentDBId={setCurrentDBId}
+                setDocumentsIds={setDocumentsIds}
               />
-              <Route
-                path="detailview"
-                element={
-                  <DetailView
-                    user={user}
-                    currentDBId={currentDBId}
-                    isEditMode={isEditMode}
-                    setIsEditMode={setIsEditMode}
-                    currentDocIndex={currentDocIndex}
-                    setCurrentDocIndex={setCurrentDocIndex}
-                    documentsIds={documentsIds}
+            )}
+            <div className="flex grow justify-center">
+              <Routes>
+                <Route path="/login" element={<Login setUser={setUser} />} />
+                <Route path="/dashboard" element={<ContentsContainer />}>
+                  <Route
+                    path="listview"
```

Comment:
문제는 아니라고 생각되지만, 문서 전반에 `<Route path="/ ...">` 이렇게 사용하고 있는 것 같아 제안드려봅니다.
```suggestion
                    path="/listview"
```